### ISSUE TITLE
Create "Movie Night" app

### ISSUE THREAD

**Issue by FalseMann** [FalseMann/nuilzero#5]
**Route**: `/movie-night`

```
// TODO: Describe how the "movie night" app should work
```

I think this depends on #3 and #4 being complete first.

**Comment by FalseMann** [comment]
@WarmongerSan were you going to look into this?

**Comment by WarmongerSan** [comment]
Yeah, I'll look into implementing the Movie Night functionality. I have an idea of how to implement it already.

**Comment by WarmongerSan** [comment]
@FalseMann I don't think we'll need any storage for now, as I feel like we don't need to save any chat data, if that makes sense? We could always add the movie picker later on when the databases are set-up and keep this one for just the stream + chat?

**Comment by FalseMann** [comment]
> @FalseMann I don't think we'll need any storage for now

I wasn't sure if the voting was going to be stored in the db. I was assuming people could suggest movies and other could upvote them.
